swarm/pss/pss_test.go

```diff
@@ -522,7 +569,7 @@ func TestNetwork(t *testing.T) {
 	} else {
 		tests = append(tests, &networkParams{
 			snapshotFile: "testdata/snapshot_8.json",
-			numMessages:  1024,
+			numMessages:  2,
```

Comment:
Because actually running these tests without hanging depends on changes @nonsense has made in `pss-networktest`. Good spot, but this will be replaced in the imminent merge with that branch.